test/link/binary_version_test.cpp

```diff
@@ -22,27 +22,20 @@ using BinaryVersion = spvtest::LinkerTest;
 TEST_F(BinaryVersion, LinkerChoosesMaxSpirvVersion) {
   spvtest::Binaries binaries = {
       {
-          SpvMagicNumber,
```

Comment:
This seems to be intentional format. Consider turning auto-format off.

Reply:
Done, though I would rather let clang-format do its thing and not have many exceptions.